demo/actions/incident_commands.py

```diff
@@ -0,0 +1,92 @@
+from response.core.models import Incident
+from response.slack.decorators.incident_command import incident_command
+from response.slack.decorators import get_help
+from response.slack.client import reference_to_id, SlackError
+from response.slack.client import SlackClient
+from datetime import datetime
+
+
+@incident_command(['help'], helptext='Display a list of commands and usage')
```

Comment:
The incident commands you've added here are part of the default code base so we don't need to duplicate them here as we'll inherit them from the Response pip package. Please can you remove these?

Reply:
Yep - removing the defaults.